helix-term/src/keymap/default.rs

```diff
@@ -26,6 +26,22 @@ pub fn default() -> HashMap<Mode, Keymap> {
         "home" => goto_line_start,
         "end" => goto_line_end,
 
+        "A-w" => { "Alter Window"
+            "A-h"|"A-left" => shrink_buffer_width,
+            "A-l"|"A-right" => grow_buffer_width,
+            "A-j"|"A-down" => shrink_buffer_height,
+            "A-k"|"A-up" => grow_buffer_height,
+            "A-f" => buffer_expand_mode,
+        },
```

Comment:
Like the `<C-w>` bindings I think there should be `<A-w><A-h>` and also `<A-w>h`. That way you don't need to keep holding alt after you've hit `<A-w>`.